src/core/server/metrics/metrics_service.ts

```diff
@@ -84,4 +93,17 @@ export class MetricsService
     }
     this.metrics$.complete();
   }
+
+  private setupOpsMetricsEventType(
+    analytics: AnalyticsServiceSetup,
+    metricsObservable: Observable<OpsMetrics>
+  ) {
+    analytics.registerEventType<OpsMetrics>({
+      eventType: 'core-ops_metrics',
+      schema: opsMetricsSchema,
+    });
+    metricsObservable.subscribe((metrics: OpsMetrics) => {
```

Comment:
NIT: should keep the subscription and unsubscribe during `stop`

Reply:
@pierre I couldn't wrap my head around how to test the subscribe/unsubscribe addition. Help 🙏  (please 😊 )

Reply:
Yea, given the Observable object is private, I'm not sure how to do either. Can't think of anything that wouldn't be hacky (access the private property from the test) or overly complicated (mock rxjs)...

I looked at other services that are unsubscribing to internal observables during `stop`, and apparently we're not testing it in any place... So I guess it's fine-ish to not test it for now.